src/main/webapp/content/scss/themes/theme-dark.scss

```diff
@@ -295,3 +295,13 @@ html {
     font-size: 0.8em;
     color: #a0aabe;
 }
+
+input:-webkit-autofill,
+input:-webkit-autofill:hover,
+input:-webkit-autofill:focus,
+input:-webkit-autofill:active {
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: white;
+    transition: background-color 5000s ease-in-out 0s;
+    box-shadow: inset 0 0 20px 20px #23232329;
```

Comment:
Where does this color come from? Is it arbitrary? Can we use a standard color here?

Reply:
It's from the [Stackoverflow answer](https://stackoverflow.com/a/14205976) from which I got the solution. I thought this would be the standard color. But now I changed it to `$input-color`, which results in no color change during autofill.